akka-typed/src/main/scala/akka/typed/receptionist/Receptionist.scala

```diff
@@ -0,0 +1,103 @@
+package akka.typed.receptionist
+
+import akka.typed.ActorRef
+import akka.typed.ActorSystem
+import akka.typed.Extension
+import akka.typed.ExtensionId
+import akka.typed.internal.receptionist.LocalReceptionist
+
+import scala.concurrent.duration._
+import scala.reflect.ClassTag
+
+class ReceptionistExt(system: ActorSystem[_]) extends Extension {
```

Comment:
I think it's ugly to name it `Ext`, since it's the public api. It can be `class Receptionist`

Reply:
Will do.